src/audienceManager.ts

```diff
@@ -45,17 +26,16 @@ export default class AudienceManager {
     constructor(
         userAudienceUrl: string,
         apiKey: string,
-        logger: SDKLoggerApi,
-        mpid: string
+        logger: SDKLoggerApi
     ) {
         this.logger = logger;
-        this.url = `https://${userAudienceUrl}${apiKey}/audience?mpid=${mpid}`;
+        this.url = `https://${userAudienceUrl}${apiKey}/audience?mpid=`;
```

Comment:
I think it makes more sense to just have the base url here and then append `?mpid=` below. It makes it more obvious as to what's going on.

```suggestion
        this.url = `https://${userAudienceUrl}${apiKey}/audience`;
```